airflow/models/variable.py

```diff
@@ -181,6 +183,7 @@ def set(
 
     @classmethod
```

Comment:
Why not @staticmethod here (As opposed to set above)?

Reply:
My bad. Thanks for pointing it out! It should be **@staticmethod**. I changed it to **@staticmethod**.

Reply:
Then also `cls` has to be removed.

Reply:
Definitely :)